src/util/keyutil.ts

```diff
@@ -114,6 +123,79 @@ class KeybindManager {
         this.keyDescriptionsMap = newKeyDescriptions;
     }
 
+    prettyPrintKeybind(keyDescription: string): string {
+        let keyPress = parseKeyDescription(keyDescription);
+        let returnString = "";
+        if (keyPress.mods.Cmd) {
+            returnString += "⌘";
+        }
+        if (keyPress.mods.Ctrl) {
+            returnString += "⌃";
+        }
+        if (keyPress.mods.Option) {
+            returnString += "⌥";
+        }
+        if (keyPress.mods.Shift) {
+            returnString += "⇧";
+        }
+        if (keyPress.mods.Meta) {
```

Comment:
How the system currently works is Cmd is automatically MetaKey on Mac and AltKey on linux and option is the opposite. But if a Keybind specifies meta it's always looking for the meta key. kind of like how you can use key code to specify a physical key, I currently have it so meta and alt allow someone to override any platform specific behavior. Does that make sense? It's definitely a niche use case but I figured there's no reason not to have it

Reply:
Also, in this case the other key would be ignored, so like if for some reason there is a keybinding that is `Meta:Cmd:c` that is interpreted as `Meta:c`, and `Alt:Option:c` is interpreted as `Option:c`. I feel like that is the most reasonable way to interpret those cases anyways